pypeit/tests/test_scripts.py

```diff
@@ -6,28 +6,80 @@
 import glob
 import shutil
 
+from configobj import ConfigObj
+
 import pytest
 
 import matplotlib
 matplotlib.use('agg')  # For Travis
 
-from pypeit import msgs
 from pypeit.scripts import setup, show_1dspec, coadd_1dspec, chk_edges, view_fits, chk_flats
```

Comment:
Can we add a test of the pypeit_trace_edges script?

Reply:
Yep, good suggestion.  Done.